modules/ROOT/pages/7.3-release-notes.adoc

```diff
@@ -29,17 +29,21 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
-=== <Premium plugin name 1>
+=== Enhanced Code Editor
 
-The {productname} {release-version} release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} {release-version} release includes an accompanying release of the **Enhanced Code Editor** premium plugin.
 
-**<Premium plugin name 1>** includes the following <fixes, changes, improvements>.
+**Enhanced Code Editor** includes the following improvements.
 
-==== <Premium plugin name 1 change 1>
+==== Added new option `advcode_prettify_getcontent` to Enhanced Code Editor.
 
-// CCFR here.
+Previously, formatting of HTML content was only applied in the **Enhanced Code Editor** view, and as a consequence, it was not possible to obtain formatted HTML content when calling the `editor.getContent` API.
```

Comment:
Both functionalities - formatted code in the `Enhanced Code Editor` view and getting formatted code on `editor.getContent` API are introduced in 7.3 so I am not sure about phrasing it like that. Sound like the first one was there before, which is not true.

Reply:
```suggestion
Previously, formatting HTML content within the **Enhanced Code Editor** view or retrieving formatted HTML from the `editor.getContent` API as not possible.
```